invenio_app_ils/ui/src/invenio_app_ils/routes/BackOffice.scss

```diff
@@ -1,3 +1,10 @@
 .container {
   padding: 1rem;
+
+  .sub.header {
```

Comment:
please be aware that this change will affect all the elements of class .header.sub

Reply:
Yes, good point! The intention was to style them all to have a unified sub header look but maybe we should style them more specifically?

Reply:
If you want to style them globally, I think it is more preferable to define something like 

`.ils-sub-header { @extend .sub.header; }`

instead of overriding the framework classes.